packages/lib-user/src/components/shared/TopProjects/TopProjects.spec.js

```diff
@@ -14,7 +14,7 @@ describe('components > UserStats > TopProjects', function () {
     expect(screen.getByText('Top Projects')).to.be.ok()
   })
 
-  it('should show a "see more" link', function () {
+  it.skip('should show a "see more" link', function () {
```

Comment:
Is there a reason this one is still getting skipped?

Reply:
Great question! Eventually there'll be an All Projects page and the "see more" link will be added back and this test can be re-enabled. I've added a comment in the test code accordingly (should have done initially 🤦‍♂️ )